app/MMB/src/context/BookingDetailContext.js

```diff
@@ -16,8 +16,10 @@ const defaultState = {
   arrivalCityId: '',
   arrivalTime: new Date(),
   departureTime: new Date(),
+  eTicketPath: null,
   actions: {
     setBookingDetail: () => {},
+    setETicketPath: () => {},
```

Comment:
Do we need to expand context with all these things? Do not create Redux mess from it, please. This doesn't sound like a context thing.

Reply:
The component tree will be like:

     ETicketPdfScreen
   |                   |
navigationHeader .     EticketPdf
   |                   |
ShareButton            PdfViewer

The only way to share this url from the Eticket-pdf-viewer with the share button in the navigation header is through a Context. I could of course create a separate context for this if you think that is better?

Reply:
OK, leave it. It's just strange that we are not in charge of the path now. I would expect we know in advance where are the PDFs going to be stored (?).

Reply:
We could do that, but now it is stored cache (tmp folder ish), we could downlaod it ourselves and probably control the path

Reply:
Only if you are super motivated to make your best here... 🙂 We can leave it like this for now.

Reply:
> We can leave it like this for now

This usually means forever 🙈 

I will have a look at it just after this is merged 👍

Reply:
> This usually means forever 🙈

Damn, this is so true... 😄

OK, thank you very much.